code/modules/client/verbs/ooc.dm

```diff
@@ -100,9 +97,12 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 	set name = "Set Your OOC Color"
 	set category = "Preferences"
 
+	/*
+	Not sure how analy retarded you made this exactly TG.
```

Comment:
>instead of unlocking everything, and locking it again later

@Shadowlight213 Never was the above a thing I said to do.
Unless you really consider giving it to admins (who already have a similar colour pref) "unlocking everything"